mcs/tools/mdoc/Test/html.expected/Mono.DocTest.Generic/Func`2.html

```diff
@@ -190,32 +190,32 @@
       <a href="../index.html">DocTest</a> : <a href="index.html">Mono.DocTest.Generic Namespace</a></div>
     <div class="SideBar">
       <p>
-        <a href="#T:Mono.DocTest.Generic.Func`2">Overview</a>
+        <a href="#T:Mono.DocTest.Generic.Func&lt;TArg,TRet&gt;">Overview</a>
```

Comment:
Even ignoring the previous point that URL-linking shouldn't contain type parameters, *why* would you choose `<` and `>` to wrap the type parameters and not `{` and `}`? The former requires escaping; the latter does not.

This would thus be a saner link:

    <a href="#T:Mono.DocTest.Generic.Func{TArg,TRet}">Overview</a>

*I don't agree that it's correct* -- I believe that the current `Func^2` link is an improvement and correct -- but using `{`/`}` is considerably saner than `<`/`>`.